### PR TITLE
For review (api)

### PR DESCRIPTION
**Pull request by stas7t**
Client app for this api [https://github.com/stas7t/todo_angular](url)

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -2,36 +2,32 @@
...
+  gem 'rails_12factor'
+end
 
 group :development, :test do
-  # Call 'byebug' anywhere in the code to stop execution and get a debugger
-  # console
   gem 'byebug', platforms: %i[mri mingw x64_mingw]
+  gem 'factory_girl_rails', '~> 4.8'
   gem 'rspec-rails', '~> 3.6'
+  gem 'sqlite3'
```
Comment (GeorgeZhukov) [inline Gemfile]: just note: for simple applications it's ok to use sqlite for development, but keep in mind that for real projects better to use ONE db backend (no matter what env is: dev, test, or production), for ex: postgresql

File: config/initializers/carrierwave.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+CarrierWave.configure do |config|
+  require 'carrierwave/orm/activerecord'
+
+  config.fog_provider = 'fog/aws'                        # required
+  config.fog_credentials = {
+    provider:              'AWS',                        # required
+    aws_access_key_id:     ENV['AWS_ACCES_KEY_ID'],      # required
```
Comment (GeorgeZhukov) [inline config/initializers/carrierwave.rb]: better to use **ENV.fetch** because it raise an error if no keys found

File: Gemfile (hunk 1 above)
Comment (stas7t) [reply Gemfile]: changed to postgresql

File: config/initializers/carrierwave.rb (hunk 2 above)
Comment (stas7t) [reply config/initializers/carrierwave.rb]: done

**Comment by stas7t** [comment]
I have added cancancan

**Pull request merged**